packages/webcert/src/feature/certificate/Buttons/RevokeDBandDoiModalContent.tsx

```diff
@@ -0,0 +1,56 @@
+import { InfoBox } from '@frontend/common'
+import _ from 'lodash'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { getCertificateMetaData, getIsLocked } from '../../../store/certificate/certificateSelectors'
+
+export const RevokeDBandDoiModalContent: React.FC = () => {
```

Comment:
Maybe doi also should be in capital letters, so it looks the same as DB? And 'and' should also start with a capital letter.
So change it to RevokeDBAndDOIModalContent